Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/adapter/component/XDRHelper.java

```diff
@@ -1,7 +1,7 @@
 /*
```

Comment:
Can we update this class  to be used for DocumentDataSubmission as well? Both DDS and DS have one common object SubmitObjectsRequest

Reply:
Updated.